understanding/22/target-size-minimum.html

```diff
@@ -52,27 +50,31 @@ <h2>Intent of Target Size (Minimum)</h2>
-            <li>If the spacing of the targets is essential to the information being conveyed, the "Essential" exception applies. For example, in digital maps, the position of pins is analogous to the position of places shown on the map. If there are many pins close together, the offset between pins and neighboring pins will often be below 24 CSS pixels. It is essential to show the pins at the correct map location, therefore the Essential exception applies. As well, some jurisdictions legally require online forms to replicate paper forms, which can impose constraints on the size of targets. In such jurisdictions, any legal requirement to replicate small targets can be considered essential.</li>
-            <li>Browsers have default renderings of some controls, such as the days of the month calendar in an <code>input type="date"</code>. As long as the author has not modified the user agent default, the target size for a "User agent control" is excepted.</li>
-            <li>In cases where a target does not have an area equivalent to 24 by 24 CSS pixels, but there is another control that can achieve the underlying function that <em>does</em> meet the minimum area, the target can be excepted based on the "Equivalent" exception.</li>
+            <li><strong>Spacing:</strong> If a target is smaller than 24 by 24 CSS pixels but has sufficient spacing, it passes. The spacing is sufficient if the offset between targets is at least 24 CSS pixels. <a href="#dfn-target-offset">Target offset</a> measures the distance between targets, measured from the farthest side of one target to the nearest side of the adjacent target. It assessed for each target, to each nearby target. What edge of the target is <em>farthest</em> depends on the relative position of the neighboring target. For example, if considering distance above the target, the farthest edge would be the bottom of the target, and the offset would be measured from this bottom edge to the bottom edge of the neighboring target above. Note that where the targets differ in size or shape, the offset between targets may also differ. The offset distance must be at least 24 CSS pixels in all cases to fall under this exception.</li>
```

Comment:
The `#dfn-target-offset` link doesn't seem to work in the rendered githack page - does there need to be an explicit anchor added to the quoted definition near the top?

Reply:
The links get 'built' when the page is published, the preview is just a quick render of the raw HTML.